en/release-notes.md

```diff
@@ -25,6 +25,96 @@ The official Python client library for MAAS is available in the Ubuntu 18.04
 [https://github.com/maas/python-libmaas/releases](https://github.com/maas/python-libmaas/releases)
 
+## 2.4.0 (alpha2)
+
+### NTP services now provided by Chrony
+
+Starting with 2.4 Alpha 2, and in common with changes being made to Ubuntu Server, MAAS replaces ‘ntpd’ with Chrony for the NTP protocol. MAAS will handle the upgrade process and automatically resume NTP service operation.
+
+### Vanilla CSS Framework Transition
+
+MAAS 2.4 is undergoing a Vanilla CSS framework transition to a new version of vanilla, which will bring a fresher look to the MAAS UI. This framework transition is currently work in progress and not all of the UI have been fully updated. Please expect to see some inconsistencies in this new release.
+
+### New Features & Improvements
+
+#### NTP services now provided by Chrony.
+
+Starting from MAAS 2.4alpha2, chrony is now the default NTP service, replacing ntpd.
+This work has been done to align with the Ubuntu Server and Security team to support
+chrony instead of ntpd. MAAS will continue to provide services exactly the same way and
+users will not be affected by the changes, handling the upgrade process transparently.
+This means that:
+
+ - MAAS will configure chrony as peers on all Region Controllers
+ - MAAS will configure chrony as a client of peers for all Rack Controllers
+ - Machines will use the Rack Controllers as they do today
+
+#### MAAS Internals optimisation
+
+MAAS 2.4 is currently undergoing major surgery to improve various areas of operation
+that are not visible to the user. These updates will improve the overall performance of
+MAAS in larger environments. These improvements include:
+
+
+ - **AsyncIO based event loop**
+    MAAS has an event loop which performs various internal actions. In older versions of MAAS, the event loop was managed by the default twisted event loop. MAAS now uses an asyncio based event loop, driven by uvloop, which is targeted at improving internal performance.
+
+ - **Improved daemon management**
+   MAAS has changed the way daemons are run to allow users to see both ‘regiond’ and ‘rackd’ as processes in the process list.
+   As part of these changes, regiond workers are now managed by a master regiond process. In older versions of MAAS each worker was directly run by systemd. The master process is now in charge of ensuring workers are running at all times, and re-spawning new workers in case of failures. This also allows users to see the worker hierarchy in the process list.
+
+
+ - **Ability to increase the number of regiond workers**
+   Following the improved way MAAS daemons are run, further internal changes have been made to allow the number of regiond workers to be increased automatically. This allows MAAS to scale to handle more internal operations in larger environments.
+   While this capability is already available, it is not yet available by default. It will become available in the following milestone release.
```

Comment:
not yet available -> not yet enabled (?)

Reply:
:+1: